wrench/src/scene.rs

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use std::collections::HashMap;
+use webrender_traits::{AuxiliaryLists, BuiltDisplayList, PipelineId, Epoch, ColorF};
+use webrender_traits::DisplayItem;
+use webrender_traits::LayerSize;
+
+/// A representation of the layout within the display port for a given document or iframe.
+#[derive(Debug)]
+pub struct ScenePipeline {
+    pub pipeline_id: PipelineId,
```

Comment:
The contents of this struct can be made private. Actually, where is it used at all? I see the `pipeline_map` getting populated, but it's only being iterated by the keys (`for pipeline_id in scene.pipeline_map.keys()`).